CRM/Core/EntityTokens.php

```diff
@@ -25,31 +32,29 @@
  * AbstractTokenSubscriber in future. It is being used to clarify
  * functionality but should NOT be used from outside of core tested code.
  */
-class CRM_Core_EntityTokens extends AbstractTokenSubscriber {
+abstract class CRM_Core_EntityTokens extends AbstractTokenSubscriber {
```

Comment:
sure- there's still an open question about merging this with AbstractTokenSubscriber or not extending it - but I'm agnostic